### PR TITLE
[TR] PIM-4429: Re-enable / re-write skip-pef scenario

### PR DESCRIPTION
**Pull request by phaseinducer**
https://akeneo.atlassian.net/browse/PIM-4429

EE PR : https://github.com/akeneo/pim-enterprise-dev/pull/929

### CODE REVIEW

**Comment by nidup** [comment]
wow, huge rework of context :+1: GTM if CI ok

**Comment by phaseinducer** [comment]
http://core-ci.akeneo.com/job/pim-pr-behat/2877/
http://core-ci.akeneo.com/job/pim-pr-behat-mongodb/967/

File: features/Context/Page/Product/Edit.php [hunk 1]
```diff
@@ -288,6 +345,11 @@ public function findField($name)
      */
     public function findFieldContainer($label)
     {
+        if (1 === preg_match('/in (.{1,3})$/', $label)) {
```
Comment (jmleroux) [inline features/Context/Page/Product/Edit.php]: Why `.{1,3}` ?

File: features/Context/Page/Product/Edit.php [hunk 2]
```diff
@@ -450,12 +518,34 @@ protected function fillTextAreaField(NodeElement $fieldContainer, $value)
...
+     *
+     * @param NodeElement $fieldContainer
+     *
+     * @return string
+     */
+    protected function getTextAreaFieldValue(NodeElement $fieldContainer)
+    {
+        $field = $fieldContainer->find('css', '.field-input textarea');
+
+        if (!$field || !$field->isVisible()) {
```
Comment (jmleroux) [inline features/Context/Page/Product/Edit.php]: `if (null === $field || !$field->isVisible())`

File: features/Context/Page/Product/Edit.php (hunk 1 above)
Comment (phaseinducer) [inline features/Context/Page/Product/Edit.php]: It's the way to detect a currency code, not the best way indeed but I just took it from elsewhere in the class.

**Comment by sumbobyboys** [comment]
GTM

**Comment by jmleroux** [comment]
GTM

**Pull request merged**